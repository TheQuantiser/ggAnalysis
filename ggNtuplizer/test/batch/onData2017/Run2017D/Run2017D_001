file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/548B6388-7737-E811-8AB7-B083FED046D8.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/ACDC5538-5E37-E811-8F79-1418774108DF.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/40BF73D9-8C37-E811-AF81-1866DAEA79D0.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/84C16AB9-7937-E811-9182-1866DAEB1FCC.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/80657F0D-6C37-E811-8518-1866DAEA6BC4.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/2C074BFE-5237-E811-BF53-B083FECFF296.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/06913A7A-8337-E811-866D-842B2B1728FF.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/3419CFAE-4A37-E811-9DA0-1866DAEA6E20.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/C0BF930E-6C37-E811-93D7-1866DAEA6C40.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/D2412012-5C37-E811-BDC3-001EC94B4F43.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/922EFF12-5C37-E811-8988-B083FED18595.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/FA8C69EB-7037-E811-A8FB-782BCB78621B.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/26D9EB81-6737-E811-914E-842B2B42B758.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/20F2A822-9C37-E811-ADA0-B083FED177B0.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/8E532E5B-5537-E811-BC8E-782BCB53A0BF.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/3E6ED91E-A337-E811-B12C-1866DAEEB0A4.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/7824A2FC-5237-E811-84F2-B083FED045EC.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/727FCC56-6537-E811-BF8F-001EC94BA3E3.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/12A99A12-6C37-E811-B2A4-B083FED179D5.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017D/SinglePhoton/MINIAOD/31Mar2018-v1/80000/12F7B80B-9837-E811-9EF2-549F3525C380.root
