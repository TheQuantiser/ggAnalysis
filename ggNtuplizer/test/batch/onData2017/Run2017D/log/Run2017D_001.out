scorpion23.spa.umn.edu
